package.json

```
@@ -45,20 +45,20 @@
   },
   "dependencies": {
     "tinymce": "^5.1.5",
-    "vl-ui-alert": "^3.0.0",
-    "vl-ui-button": "^4.0.0",
-    "vl-ui-core": "^5.0.1",
-    "vl-ui-icon": "^4.0.0",
-    "vl-ui-toaster": "^3.0.0",
+    "vl-ui-alert": "^3.0.1",
+    "vl-ui-button": "github:milieuinfo/webcomponent-vl-ui-button#E2E_fix_getIcon",
```

```suggestion
    "vl-ui-button": "^5.0.1",
```
Eerst button nog releasen